content/en/user-guide/integrations/cloud-custodian/index.md

```diff
@@ -0,0 +1,117 @@
+2023-10-16T15:27:35.504  INFO --- [   asgi_gw_0] l.u.container_networking   : Determined main container target IP: 172.17.0.2
+2023-10-16T15:27:36.154  INFO --- [   asgi_gw_0] l.s.ec2.vmmanager.docker   : Instance i-d87f1ab75e95ab0d2 will be accessible via SSH at: 127.0.0.1:22, 172.17.0.3:22
+2023-10-16T15:27:36.155  INFO --- [   asgi_gw_0] l.s.ec2.vmmanager.docker   : Instance i-d87f1ab75e95ab0d2 port mappings (container -> host): {'22/tcp': 22}
+2023-10-16T15:27:36.218  INFO --- [   asgi_gw_0] localstack.request.aws     : AWS ec2.RunInstances => 200
+```
+
+### Create a Cloud Custodian policy
+
+You can now create a Cloud Custodian policy to stop the EC2 instances with the tag `Custodian`. Create a file named `custodian.yml` and add the following content:
+
+```yaml
+policies:
+  - name: my-first-policy
+    resource: aws.ec2
+    filters:
+      - "tag:Custodian": present
+    actions:
+      - stop
+```
+
+The above policy specifies the following:
+
+- `name`: The name of the policy.
+- `resource`: The AWS resource to apply the policy to.
+- `filters`: The filters to apply to the resource. In this case, the filter is `tag:Custodian` and the value is `present`.
+- `actions`: The actions to apply to the resource. In this case, the action is `stop`.
+
+### Run the Cloud Custodian policy
+
+You can now run the Cloud Custodian policy using the following command:
+
+{{< command >}}
+$ custodian run \
+    --output-dir=. custodian.yml \
+    --profile localstack
+{{< / command >}}
+
+You should see the following output:
+
+```sh
+2023-10-16 21:06:13,853: custodian.policy:INFO policy:my-first-policy resource:aws.ec2 region:us-east-1 count:1 time:0.20
+2023-10-16 21:06:13,961: custodian.policy:INFO policy:my-first-policy action:stop resources:1 execution_time:0.10
+```
+
+You can navigate to the LocalStack logs to verify that the EC2 instance was stopped successfully:
+
+```bash
+2023-10-16T15:36:13.583  INFO --- [   asgi_gw_0] localstack.request.aws     : AWS sts.GetCallerIdentity => 200
+2023-10-16T15:36:13.851  INFO --- [   asgi_gw_1] localstack.request.aws     : AWS ec2.DescribeInstances => 200
+2023-10-16T15:36:13.960  INFO --- [   asgi_gw_0] localstack.request.aws     : AWS ec2.StopInstances => 200
+```
+
+### Create CloudWatch metrics for Cloud Custodian
+
+Cloud Custodian creates CloudWatch metrics for each policy. These metrics show how many resources met the filters, the time taken to gather and filter those resources, and the time required to perform actions.
+
+Certain filters and actions might produce their own metrics. To activate metric output, you must set the `metrics` flag running Cloud Custodian.
+
+{{< command >}}
+$ custodian run -s . \
+    --metrics aws custodian.yml \
+    --profile localstack
+{{< / command >}}
+
+You can access the CloudWatch metrics over the [LocalStack Web Application](https://app.localstack.cloud/inst/default/resources/cloudwatch).
```

Comment:
nit: Not sure about the preposition `over` in this context - 🤔  perhaps "**in** the ..." or "**via** the ..." would be more common here? (we could also get feedback from a native speaker in this case, though.. ;) )
```suggestion
You can access the CloudWatch metrics in the [LocalStack Web Application](https://app.localstack.cloud/inst/default/resources/cloudwatch).
```